plugins/pipelines/metrics.js

```diff
@@ -58,6 +70,22 @@ module.exports = () => ({
                         config.aggregateInterval = aggregateInterval;
                     }
 
+                    // Format downtimeJobs and downtimeStatuses and pass them in
+                    const downtimeJobs = request.query[DOWNTIME_JOBS_KEY];
+                    const downtimeStatuses = request.query[DOWNTIME_STATUSES_KEY];
+
+                    if (downtimeJobs) {
+                        config.downtimeJobs = Array.isArray(downtimeJobs)
```

Comment:
I think `downtimeJobs` & `downtimeJobs` will always be an array. Otherwise, the 
`request.query['downtimeJobs[]'];` would have to be `request.query['downtimeJobs'];` for non-array case.

And it's good you check here explicitly.

Reply:
Actually, I think you might need this `https://www.npmjs.com/package/jquery-deparam` to take care of the case of nested arrays. Like deserializing the last one.

![image](https://user-images.githubusercontent.com/15989893/97223606-3d3a7180-178d-11eb-9a97-9ac7fae210f0.png)

Reply:
For this call `GET /pipelines/123/metrics?downtimeJobs[]=123&downtimeJobs[]=456&downtimeStatuses[]=ABORTED`
 The `request.query` looks like this:
```
{
  'downtimeJobs[]': [ '123', '456' ],
  'downtimeStatuses[]': 'ABORTED',
  sort: 'descending'
}
```